src/api/CommunicationAPI.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Fetches the data of the users messages given user id
+ * @param {number} userId id of the user
+ * @return {Promise} data of the users messages
+ */
+export const getMessages = async (userId) => {
+  console.log(userId);
+  return [
+    {
+      sender: 'senderid',
+      receiver: 'receiverid',
+      timestamp: 1647383521,
+      content: 'test message 1',
+    },
+    {
+      sender: 'receiverid',
+      receiver: 'receiverid',
+      timestamp: 1647383522,
+      content: 'test message 2',
+    },
+    {
+      sender: 'senderid',
+      receiver: 'receiverid',
+      timestamp: 1647383523,
+      content: 'test message 3',
+    },
+  ];
+};
+
+/**
+ * Sends a message from one user to another
+ * @param {string} sender id of the sender
+ * @param {string} receiver id of the receiver
+ * @param {string} content content of the message
+ * @return {Promise} data entry of the message sent
+ */
+export const sendMessage = async ({ sender, receiver, content }) => {
+  const timestamp = Date.now();
+  console.log(sender, receiver, content, timestamp);
+};
```

Comment:
While we (Comms team) are able to implement this endpoint if desired, we were thinking a SocketIO event would suffice. See https://github.com/SE701-T1/backend/pull/72

Reply:
Think this was added cause it exists in the backend communication controller. I'm happy to remove this, we just need to deprecate or remove the endpoint in the backend.

Reply:
Right that makes sense. My PR does remove it from the controller :+1:

Reply:
Sure I've removed it